src/applications/appeals/995/containers/App.jsx

```diff
@@ -140,26 +146,14 @@ export const App = ({
   );
 
   // Go to start page if we don't have an expected benefit type
-  if (!location.pathname.endsWith('/start') && !hasSupportedBenefitType) {
+  if (!pathname.endsWith('/start') && !hasSupportedBenefitType) {
     router.push('/start');
     content = wrapInH1(
       <va-loading-indicator
         set-focus
         message="Please wait while we restart the application for you."
       />,
     );
-  } else if (
-    loggedIn &&
-    hasSupportedBenefitType &&
-    ((contestableIssues.status || '') === '' ||
-      contestableIssues.status === FETCH_CONTESTABLE_ISSUES_INIT)
-  ) {
-    content = wrapInH1(
-      <va-loading-indicator
-        set-focus
-        message="Loading your previous decisions..."
-      />,
-    );
```

Comment:
Removed because this was causing the e2e tests to fail, and there are 2 other API calls happening at the same time: in progress data loading & ITF check.